app/models/external_events/external_policy_member_drop.rb

```diff
@@ -88,12 +88,17 @@ def extract_rating_details
       end
     end
 
+    def extract_rating_tier
+      p_enrollment = Maybe.new(@total_source).policy_enrollment.value
```

Comment:
This is line where the new policy is being used instead of the terminated policy.